README.md

```diff
@@ -2,7 +2,7 @@
 
 <p align='center'>
   <a href='https://zap.jackmallers.com'>
-    <img src='https://i.imgur.com/O1m9wMP.png' alt='screenshot' />
+    <img src='https://imgur.com/sfmOvhq' alt='screenshot' />
```

Comment:
I think this needs to be `https://i.imgur.com/sfmOvhq.jpg` (it's missing `.jpg`)

Reply:
or `.png`